lib/buffer.js

```diff
@@ -237,6 +237,17 @@ Buffer.from = function from(value, encodingOrOffset, length) {
   );
 };
 
+// Identical to the built-in %TypedArray%.of(), but avoids using the deprecated
+// Buffer() constructor.
+//
+// Refs: https://tc39.github.io/ecma262/#sec-%typedarray%.of
+Buffer.of = (...items) => {
+  const newObj = allocate(items.length);
```

Comment:
Hm … we may want `new FastBuffer(items.length)` here, so that the returned buffer is not pooled, like you’d expect from the standard TypedArray methods.

Also, I’m curious, do you know why `TypedArray.of` does not use `this[Symbol.species]`? That seems like a good fix for this problem in the long run.

Reply:
Is it different from `Buffer.from(items)`?

Reply:
Yes; `Buffer.of` is `(...items)`.

Reply:
That's already in the header, but is the implementation here different in behaviour from just returning `Buffer.from(items)`? Do we need to reimplement it?